modules/nf-core/quartonotebook/main.nf

```diff
@@ -0,0 +1,97 @@
+include { dumpParamsYaml; indentCodeBlock } from "./parametrize"
+
+process QUARTONOTEBOOK {
+    tag "$meta.id"
+    label 'process_low'
+
+    // NB: You'll likely want to override this with a container containing all
+    // required dependencies for your analyses. You'll at least need Quarto
+    // itself, Papermill and whatever language you are running your analyses on;
+    // you can see an example in this module's Dockerfile.
+    container "docker.io/erikfas/quartonotebook"
+
+    input:
+    tuple val(meta), path(notebook)
+    val parameters
+    path input_files
+
+    output:
+    tuple val(meta), path("*.html")     , emit: html
+    tuple val(meta), path("artifacts/*"), emit: artifacts, optional: true
+    tuple val(meta), path("params.yml") , emit: params_yaml, optional: true
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "The QUARTONOTEBOOK module does not support Conda/Mamba, please use Docker / Singularity / Podman instead."
+    }
```

Comment:
why? I think quarto is on conda-forge?

Reply:
It's complicated 😅 I wrote a bit about this in commit [6c9a653](https://github.com/nf-core/modules/pull/4876/commits/6c9a653ca1f02a3dd4ab0d5996cbc5726b82a90e), but in essence it's because of several problems with architecture differences and Pandoc. Quarto is indeed on conda-forge (albeit an outdated version), but this does not work on ARM64 due to Pandoc version incompatibilities. Manually installing Quarto from their `tar.gz` releases manually inside a Docker container works for both AMD64 and ARM64, though, so that means the module will work on either architecture.

The problem comes with having a Conda directive with a different environment than the container-based one; the difference being in how Quarto and its dependencies are installed. I did retain the Conda directive (plus a disallowance if-clause for ARM64 systems) until the previously mentioned commit, but the snapshot for testing always turned out different than the container-based environment (even with the same Quarto version installed).

So, ideally, Conda will at some point in the future update their Quarto and Pandoc versions such that the dependency tree can be solved for ARM64 systems, at which point we can add the Conda directive back in.

Reply:
I see, would be great to add a short comment in the code for future reference.

Reply:
Good point, I have added this now.